src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+
+Cela dit, les requêtes de polices de caractères faites sur les appareils mobiles sont légèrement plus nombreuses. J'ai l'impression que moins de polices sont disponibles sur les appareils mobiles, ce qui signifie moins d'occurrences `local()` dans les CSS de Google Fonts, et donc moins de requêtes sur le réseau pour ces polices.
+
+### Vous ne voulez pas gagner ce prix
+
+{{ figure_markup(
+  caption="Le plus grand nombre de demandes de polices Web sur une seule page.",
+  content="718",
+  classes="big-number"
+)
+}}
+
+Le prix de la page qui demande le plus de polices web est attribué à un site qui a fait **718** requêtes de polices web !
+
+Après avoir plongé dans le code, toutes ces 718 requêtes vont à Google Fonts ! Il semble qu'un plugin d'optimisation "Above the Page fold" pour WordPress qui ne fonctionne pas bien soit devenu défectueux sur ce site et demande (DDoS-ing ?) toutes les polices Google !
+
+Il est ironique qu'un plugin d'optimisation des performances puisse rendre vos performances bien pires !
+
+## Une correspondance plus précise avec `unicode-range`.
+
+{{ figure_markup(
+  caption="Pourcentage des pages mobiles qui déclarent une police web avec la propriété <code>unicode-range</code>.",
+  content="56%",
+  classes="big-number"
+)
+}}
+
+[`unicode-range`](https://developer.mozilla.org/fr/docs/Web/CSS/%40font-face/unicode-range) est une excellente propriété CSS qui permet au navigateur de savoir précisément quels points de code la page souhaite utiliser dans le fichier de police. Si la déclaration `@font-face` a une "plage de codes"(`unicode-range`), le contenu de la page doit correspondre à l'un des points de code de la plage avant que la police ne soit demandée. C'est une très bonne chose.
+
+C'est une autre mesure qui, je pense, a été faussée par l'utilisation des Google Fonts, car Google Fonts utilise la "plage unicode" (`unicode-range`) dans la plupart (sinon la totalité) de ses CSS. Je m'attendrais à ce que cela soit moins fréquent du côté des utilisateurs, mais peut-être qu'il sera possible de filtrer les demandes de Google Fonts dans la prochaine édition de l'Almanach.
+
+## Ne demandez pas de police web si la police système existe
+
+{{ figure_markup(
+  caption="Pourcentage de pages mobile qui déclarent une police web avec la propriété <code>local()</code>.",
+  content="59%",
+  classes="big-number"
+)
+}}
+
+`local()` est une belle façon de référencer une police système dans votre `@font-face` `src`. Si la police `local()` existe, il n'est pas du tout nécessaire de faire une requête pour une police web. Cette propriété est utilisée de manière extensive et controversée par Google Fonts, et il s'agit donc probablement d'un autre exemple de données biaisées si nous essayons de glaner des modèles à partir du terrain des utilisateurs. - du côté des utilisateurs
```

Comment:
`à partir du terrain des utilisateurs. - du côté des utilisateurs` ??? (un oubli ?)

Reply:
I['m not sure what you mean here. Could you suggest the change needed?

Reply:
```suggestion
`local()` est une belle façon de référencer une police système dans votre `@font-face` `src`. Si la police `local()` existe, il n'est pas du tout nécessaire de faire une requête pour une police web. Cette propriété est utilisée de manière extensive et controversée par Google Fonts, et il s'agit donc probablement d'un autre exemple de données biaisées si nous essayons de glaner des modèles du côté des utilisateurs.

```

Reply:
Updated in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2114/commits/5e0ba5fc770ce69bb3edb46a52ba42f961d1e89b